pkg/metadata/common/common_windows.go

```diff
@@ -0,0 +1,44 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+// +build windows
+
+package common
+
+import (
+	"strings"
+	"syscall"
+	"unsafe"
+)
+
+var getUUID = GetUUID
+
+// GetUUID returns the machine GUID on windows; copied from gopsutil
+func GetUUID() string {
```

Comment:
Could we add the `cache` part like for the linux code. Maybe do we want to have a common function that check the cache and two functions (for linux and windows) to fetch the UUID ?